data-prepper-core/integrationTest.gradle

```diff
@@ -68,24 +68,54 @@ task buildDataPrepperDockerImage(type: DockerBuildImage) {
     images.add("integ-test-pipeline-image")
 }
 
-task createDataPrepperDockerContainer(type: DockerCreateContainer) {
+task createDataPrepper1DockerContainer(type: DockerCreateContainer) {
     dependsOn buildDataPrepperDockerImage
     dependsOn createDataPrepperNetwork
+    containerName = "data-prepper1"
     hostConfig.portBindings = ['21890:21890']
     hostConfig.network = createDataPrepperNetwork.getNetworkName()
     targetImageId buildDataPrepperDockerImage.getImageId()
 }
 
-task startDataPrepperDockerContainer(type: DockerStartContainer) {
-    dependsOn createDataPrepperDockerContainer
-    targetContainerId createDataPrepperDockerContainer.getContainerId()
+task createDataPrepper2DockerContainer(type: DockerCreateContainer) {
+    dependsOn buildDataPrepperDockerImage
+    dependsOn createDataPrepperNetwork
+    containerName = "data-prepper2"
+    hostConfig.portBindings = ['21891:21890']
+    hostConfig.network = createDataPrepperNetwork.getNetworkName()
+    targetImageId buildDataPrepperDockerImage.getImageId()
+}
+
+task startDataPrepper1DockerContainer(type: DockerStartContainer) {
+    dependsOn createDataPrepper1DockerContainer
+    targetContainerId createDataPrepper1DockerContainer.getContainerId()
     doLast {
         sleep(10*1000)
     }
 }
 
-task stopDataPrepperDockerContainer(type: DockerStopContainer) {
-    targetContainerId createDataPrepperDockerContainer.getContainerId()
+task startDataPrepper2DockerContainer(type: DockerStartContainer) {
+    dependsOn createDataPrepper2DockerContainer
+    targetContainerId createDataPrepper2DockerContainer.getContainerId()
+    doLast {
+        sleep(10*1000)
+    }
+}
+
+task stopDataPrepper1DockerContainer(type: DockerStopContainer) {
+    targetContainerId createDataPrepper1DockerContainer.getContainerId()
+}
+
+task stopDataPrepper2DockerContainer(type: DockerStopContainer) {
```

Comment:
Nit: This works for now, but we may want to look into a way of calling a task with an argument if we can? For example:

startDataPrepperContainers(int[] ports)

That way we can scale up testing and re-use these functions

Reply:
Agreed. We probably need to define new classes in gradle for this purpose. For the purpose of this test it is not worth the efforts. Will re-examine when it becomes necessary.